lib/rex/proto/kerberos/credential_cache/krb5_ccache_presenter.rb

```diff
@@ -106,7 +106,7 @@ def present_cred(cred, key: nil)
       output.join("\n")
     end
 
-    # @param [Rex::Proto::Kerberos::Pac::Krb5LogonInfo] logon_info
+    # @param [Rex::Proto::Kerberos::Pac::Krb5ValidationInfo] logon_info
```

Comment:
Does this change the expected type?

Reply:
https://github.com/rapid7/metasploit-framework/blob/782e4c0295b1845f47fec4a3c86eef29e78ac9dd/lib/rex/proto/kerberos/pac/krb5_pac.rb#L268
Looks like I typo'd the original should be `Krb5LogonInformation`

Reply:
Technically it's a `Rex::Proto::Kerberos::Pac::Krb5PacElement`

```
[6] pry(#<Rex::Proto::Kerberos::CredentialCache::Krb5CcachePresenter>)> logon_info.class
=> Rex::Proto::Kerberos::Pac::Krb5PacElement
[5] pry(#<Rex::Proto::Kerberos::CredentialCache::Krb5CcachePresenter>)> logon_info.is_a?(Rex::Proto::Kerberos::Pac::Krb5LogonInformation)
=> false
```

But I'll update it to be `Rex::Proto::Kerberos::Pac::Krb5LogonInformation`